README.md

```diff
@@ -1,10 +1,10 @@
-[![Jenkins Coverage](https://img.shields.io/jenkins/coverage/jacoco?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fiudx%2520file-server%2520%28master%29%2520pipeline%2F)](https://jenkins.iudx.io/job/iudx%20file-server%20(master)%20pipeline/lastBuild/jacoco/)
-[![Unit Tests](https://img.shields.io/jenkins/tests?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fiudx%2520file-server%2520%28master%29%2520pipeline%2F)](https://jenkins.iudx.io/job/iudx%20file-server%20(master)%20pipeline/lastBuild/testReport/)
-[![Performance Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fiudx%2520file-server%2520%28master%29%2520pipeline%2F&label=performance%20tests)](https://jenkins.iudx.io/job/iudx%20file-server%20(master)%20pipeline/lastBuild/performance/)
-[![Security Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fiudx%2520file-server%2520%28master%29%2520pipeline%2F&label=security%20tests)](https://jenkins.iudx.io/job/iudx%20file-server%20(master)%20pipeline/lastBuild/zap/)
-[![Integration Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fiudx%2520file-server%2520%28master%29%2520pipeline%2F&label=integration%20tests)](https://jenkins.iudx.io/job/iudx%20file-server%20(master)%20pipeline/HTML_20Report/)
+[![Build Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520file-server%2520(v5.5.0)%2520pipeline%2F)](https://jenkins.iudx.io/job/iudx%20file-server%20(v5.5.0)%20pipeline/lastBuild/)
+[![Jenkins Coverage](https://img.shields.io/jenkins/coverage/jacoco?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520file-server%2520(v5.5.0)%2520pipeline%2F)](https://jenkins.iudx.io/job/iudx%20file-server%20(v5.5.0)%20pipeline/lastBuild/jacoco/)
+[![Unit Tests](https://img.shields.io/jenkins/tests?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520file-server%2520(v5.5.0)%2520pipeline%2F)](https://jenkins.iudx.io/job/iudx%20file-server%20(v5.5.0)%20pipeline/lastBuild/testReport/)
+[![Performance Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520file-server%2520(v5.5.0)%2520pipeline%2F&label=performance%20tests)](https://jenkins.iudx.io/job/iudx%20file-server%20(v5.5.0)%20pipeline/lastBuild/performance/)
+[![Security Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520file-server%2520(v5.5.0)%2520pipeline%2F&label=security%20tests)](https://jenkins.iudx.io/job/iudx%20file-server%20(v5.5.0)%20pipeline/lastBuild/zap/)
+[![Integration Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520file-server%2520(v5.5.0)%2520pipeline%2F&label=integration%20tests)](https://jenkins.iudx.io/job/iudx%20file-server%20(v5.5.0)%20pipeline/lastBuild/HTML_20Report/)
```

Comment:
See if unit tests and integration tests are within same report and use single badge for both

Reply:
added single badge for unit and integration tests!